aang7-Monterrey.md

```diff
@@ -0,0 +1,4 @@
+## Monterrey, NL
+
```

Comment:
@aang7 I'm from California :sunny: . Anything else I should know when visiting NL?